.eslintrc

```diff
@@ -0,0 +1,30 @@
+{
+  "extends": "xo-space",
+  "rules": {
+    "strict": [2, "global"],
+    "no-var": 1
```

Comment:
Have set this to warning (not error). There is a nice description [here](http://eslint.org/docs/rules/no-var). Any problems with this?